package.json

```diff
@@ -8,8 +8,7 @@
     "compile": "tsc --build tsconfig.build.json",
     "compile:clean": "npm run compile -- --clean",
     "watch": "npm run compile && tsc --build tsconfig.json --watch",
-    "version": "npm i --package-lock-only && node ./prompt-update-changelogs.js",
-    "release:version-bump": "lerna version",
+    "version": "npm i --package-lock-only && git add package-lock.json",
```

Comment:
I added this prompt awhile back. I totally don't mind seeing it be removed, but the main advantage here was landing CHANGELOG updates inside the same release commit that lerna made which I found to be nice. This can only be done during the `version` step (as you've noticed, Lerna calls this script as part of the lifecycle).

It never was a big deal - we did tons of releases with them being separate commits, but just pointing it out in case you think "oh yeah I actually prefer that".

Reply:
I do like landing those changelogs in the same commit. I think I was a little confused being prompted to make CHANGELOG changes after the instructions in `DEVELOPMENT.md` said to open a PR with the CHANGELOG changes.

I don't feel strongly about it, as long as we can keep the instructions fairly straightforward.

Reply:
Ah, that's on me for not updating the instructions then. Very open to other approaches here as well if you've got anything in mind. My only real motivation here was to afford the user some time during the `version` hook where the CHANGELOGs could be updated.

Reply:
I removed the prompt for now, we'll see how that goes